crates/cxx-qt-gen/src/parser/qobject.rs

```diff
@@ -46,6 +44,19 @@ pub struct ParsedQObject {
 }
 
 impl ParsedQObject {
+    const ALLOWED_ATTRS: [&'static str; 11] = [
+        "cxx_name",
+        "rust_name",
+        "namespace",
+        "derive",
```

Comment:
what is `derive` used for on the QObject? And would that even work? As wouldn't it be on the Rust struct side not the QObject type in the extern "RustQt" ?

Reply:
#[derive(Default)] for construction, It was in some tests or examples somewhere I believe

Reply:
Hmm, i think most of those would be on the Rust struct?

```rust
#[cxx_qt::bridge]
mod ffi {
   extern "RustQt" {
     #[qobject]  // there is no derive here
      type MyObject = super::MyObjectRust;
   }
}

#[derive(Default)]   // this struct must implement Default for the default constructor
struct MyObjectRust;
```

I think maybe those are the derive on the struct outside of the bridge? So i think it's likely we don't need derive here ? :thinking:

Reply:
ahh it was just in some of the test_inputs, and removing them was fine so I've done that